### ISSUE TITLE
Value Function Normalization

### ISSUE THREAD

**Issue by huvar** [hill-a/stable-baselines#892]
Hello,

I was reading through the paper "What Matters In On-Policy Reinforcement Learning? A Large-Scale Empirical Study" and from their recommendations there are two implementation improvements which are currently not available: 
1. Value function normalization - I would say that this normalization could be implemented in the runner. I am not sure if this improvement is redundant when the rewards are normalized.
2. Recompute advantages at the beginning of each epoch - I am not sure if I got this right... If I understand correctly one needs to keep observations to recalculate the value functions and advantages which seems a heavy work (specially considering recurrent policies). Do I get it right?

What do you think? Do these improvements make sense?

**Comment by Miffyli** [comment]
I agree that we should at least try these suggestions by the authors of said paper, but I think these should be left for stable-baselines3. This repository is mainly for bug-fixing now, and focus is on stable-baselines3.

Another view: I like the paper as much as the next guy, but we should remember that the experiments were only done on _continuous_ tasks in the _MuJoCo-like envs_, and as such we should not immediately take it for granted that these tricks are equally beneficial to other envs (e.g. Atari). I think authors mentioned something along these lines in the paper, too.
